src/main/scala/dx/exec/TaskRunner.scala

```diff
@@ -81,19 +81,19 @@ case class TaskRunner(task: TAT.Task,
         }
       case _ => throw new Exception("Malformed environment serialized to disk")
     }
-    val localizedInputs = locInputsM.map {
+    val localizedInputs = localPathToJs.map {
       case (key, JsArray(Vector(JsString(wdlTypeRepr), jsVal))) =>
         val t = TypeSerialization(typeAliases).fromString(wdlTypeRepr)
         val value = WdlValueSerialization(typeAliases).fromJSON(jsVal)
         key -> (t, value)
       case (_, other) =>
         throw new Exception(s"Bad deserialization value ${other}")
     }
-    val dxUrl2path = dxUrlM.map {
-      case (key, JsString(path)) => Furl.fromUrl(key, dxApi) -> Paths.get(path)
-      case other                 => throw new Exception(s"Invalid map item ${other}")
+    val fileSourceToPath = dxUriToJs.map {
+      case (uri, JsString(path)) => fileResolver.resolve(uri) -> Paths.get(path)
+      case (_, _)                => throw new Exception("Sanity")
```

Comment:
Didn't we want to remove the "sanity" from our vocab?

Reply:
yep, I just missed this one when merging